source/grid/tria.cc

```diff
@@ -849,9 +849,9 @@ namespace internal
     else
 #endif
       {
-        (void)global_first_cell;
-        (void)global_num_cells;
-        (void)mpi_communicator;
+        // (void)global_first_cell;
+        // (void)global_num_cells;
+        // (void)mpi_communicator;
```

Comment:
Instead of commenting these lines you can remove them :).
```suggestion
```

Reply:
By way of explanation, what @blaisb is saying is that there is no value in keeping these lines around: Why would anyone want to know that these lines were once part of the source code? So if there is no value in keeping them, let's just remove them!